src/dep/mod.rs

```diff
@@ -60,26 +60,67 @@ fn get_component_normal(component: Component) -> Option<&OsStr> {
     }
 }
 
-#[cfg(all(test, unix))]
+#[cfg(test)]
 mod tests {
+    use std::path::PathBuf;
+
     use super::*;
 
     #[test]
     fn from_std_path_returns_correct_variant() {
-        let cratesio = StdPath::new(
-            "/home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/cortex-m-rt-0.6.13/src/lib.rs",
-        );
-        let rustc = StdPath::new(
-            "/rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/panicking.rs",
-        );
-        let rust_std = StdPath::new(
-            "/home/user/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/sync/atomic.rs",
-        );
-        let local = StdPath::new("src/lib.rs");
+        let home = dirs::home_dir().unwrap();
+        let home = home.to_str().unwrap();
+
+        let cratesio = PathBuf::new()
+            .join(home)
+            .join(".cargo")
+            .join("registry")
+            .join("src")
+            .join("github.com-1ecc6299db9ec823")
+            .join("cortex-m-rt-0.6.13")
+            .join("src")
+            .join("lib.rs");
+        assert!(matches!(
+            Path::from_std_path(cratesio.as_path()),
+            Path::Cratesio(_)
+        ));
```

Comment:
```suggestion
        assert!(matches!(Path::from_std_path(&cratesio), Path::Cratesio(_)));
```

Reply:
Rust-fmt spans the `assert!` over multiple lines.

Reply:
For me it autoformatted it into one line after replacing the `.as_path()` with `&`